website/content/docs/nia/architecture.mdx

```diff
@@ -53,8 +45,35 @@ network infrastructure. The following [drivers](/docs/nia/network-drivers#terraf
 
 Each driver includes a set of providers that [enables support](/docs/nia/terraform-modules) for a wide variety of infrastructure applications.
 
-## Security Guidelines
+## State storage and persistence  
+
+The following types of state information are associated with CTS.
+
+### Terraform state information 
+
+By default, CTS stores [Terraform state data](https://www.terraform.io/docs/state/index.html) in the Consul KV, but you can specify where this information is stored by configuring the `backend` setting in the [Terraform driver configuration](/docs/nia/configuration#backend). If the backend is not configured to a local location, then the data persists if CTS stops.  
```

Comment:
I see the logic, but I think this is a case with two condition because we don't expect CTS to stop, AFAIK. It might be possible that there are more than two possible locations (remote, local, ?), so this still might not be accurate.